interfaces/cython/cantera/test/test_purefluid.py

```diff
@@ -149,11 +149,18 @@ def test_fd_properties_twophase(self):
 
     def test_TPX(self):
         self.water.TX = 400, 0.8
-        T,P,X = self.water.TPX
+        T, P, X = self.water.TPX
         self.assertNear(T, 400)
         self.assertNear(X, 0.8)
-        with self.assertRaises(AttributeError):
-            self.water.TPX = 500, 101325, 0.3
+
+        self.water.TPX = T, P, X
+        self.assertNear(X, 0.8)
+        with self.assertRaises(ValueError):
```

Comment:
Since these `ValueError`s are different (at least, the third is different from the first two), can you use `assertRaisesRegex` to check the message in addition to the type?